x-pack/plugins/security_solution/public/resolver/store/data/reducer.test.ts

```diff
@@ -167,6 +169,15 @@ describe('Resolver Data Middleware', () => {
 
         expect(selectedEventsForFirstChildNode).toBe(firstChildNodeInTree.relatedEvents);
       });
+      it('should return an overall correct count for the number of related events', () => {
+        const aggregateTotalByEntityId = selectors.relatedEventAggregateTotalByEntityId(
+          store.getState()
+        );
+        const countForId = aggregateTotalByEntityId(firstChildNodeInTree.id);
+        expect(countForId).toBe(
+          aggregateCategoryTotalForFirstChildNode + 1 /* +1 for the overcounted category*/
```

Comment:
what's the over-counted category? Think I missed something here

Reply:
Cool, just read the comment below. Not necessary, but could help to make an addition like `'file' + ['network', 'dns']` yields a count of 3

Reply:
1 "fake" one is added to test the limit warning (see: https://github.com/elastic/kibana/pull/76195/files#diff-b6a402f88c9d8a056133dab8c5438dd7R205 )

Maybe I should move this to one of the other describes so we don't have to do that.

Reply:
Took it here. Thanks! https://github.com/elastic/kibana/pull/76195/commits/41007c166afb524a7e47f78b05b8fbe067bfae66

Reply:
Thank you!